css/style.css

```diff
@@ -0,0 +1,36 @@
+* {
+  font-family: "Segoe UI", "Roboto", "Helvetica Neue", Arial, sans-serif;
+  font-size: 16px;
+}
+
+#main {
+  background-color: #eee;
+  padding: 15px;
+  margin: 30px 8px;
+  font-size: 1.6rem;
+  line-height: 1.5;
+  text-align: center;
+  color: #333;
+}
```

Comment:
[optional] here you can use padding for the main section from left and right, to have a nice design